jenkins-scripts/docker/lib/debbuild-base.bash

```diff
@@ -183,7 +183,14 @@ echo '# END SECTION'
 fi
 
 echo '# BEGIN SECTION: create source package' \${OSRF_VERSION}
-debuild --no-tgz-check -uc -us -S --source-option=--include-binaries
+
+# lintian triggers a problem on arm in Focal when using qemu, avoid it
+no_lintian_param=""
+if [[ ${DISTRO} == 'focal' && ${ARCH} == 'arm64' ]]; then
```

Comment:
an armhf build is hanging too:

* https://build.osrfoundation.org/job/gazebo9-debbuilder/587/console